src/platform_impl/windows/event_loop.rs

```diff
@@ -648,6 +662,17 @@ pub(crate) fn subclass_window<T>(window: HWND, subclass_input: SubclassInput<T>)
     assert_eq!(subclass_result, 1);
 }
 
+fn remove_window_subclass<T>(window: HWND, _subclass_input: Box<SubclassInput<T>>) {
+    let removal_result = unsafe {
+        commctrl::RemoveWindowSubclass(
+            window,
+            Some(public_window_callback::<T>),
+            WINDOW_SUBCLASS_ID,
+        )
+    };
+    assert_eq!(removal_result, 1);
```

Comment:
I'm not sure how I feel about these assertions. I feel like the most common way for this to fail is if the subclass is already gone and I'm not sure if we should tear down the entire application in that case.

With applications running multiple windows this could lead to a crash of the entire application, assuming this subclass info is added separately for each window (I'm not sure about that?).

Reply:
The subclass is registered individually for every window, so I don't see how it could be "already gone" when `event_loop` is the only module with enough information (the callback function pointer and subclass ID) to do so.

Reply:
Additionally, it seems like the only "always safe" place to deregister subclasses is during window destruction.